numpy/core/_methods.py

```diff
@@ -189,8 +189,22 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     # Note that x may not be inexact and that we need it to be an array,
     # not a scalar.
     x = asanyarray(arr - arrmean)
+
+    # Complex types to -> (2,)float view for fast-path
+    _complex_to_float = {
+        nt.complex64 : nt.float32,
+        nt.complex128 : nt.float64,
+        nt.clongdouble: nt.longdouble,
+    }
     if issubclass(arr.dtype.type, (nt.floating, nt.integer)):
         x = um.multiply(x, x, out=x)
+    # Fast-paths for built-in complex types with native byte order
+    elif x.dtype.type in _complex_to_float and arr.dtype.isnative:
+        xv = x.view(dtype=(_complex_to_float[x.dtype.type], (2,)))
```

Comment:
With my dict suggestion, this becomes
```suggestion
    elif x.dtype in _complex_to_float:
        xv = x.view(dtype=(_complex_to_float[x.dtype], (2,)))
```